<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegDialog</class>
 <widget class="QDialog" name="RegDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>角色性别：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEditID"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_PassWD1"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>确认密码：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit_Name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>输入密码：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>角色大名：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_PassWD2"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>输入账号：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
